test/e2e/specs/editor/blocks/navigation-frontend-interactivity.spec.js

```diff
@@ -66,10 +66,14 @@ test.describe( 'Navigation block - Frontend interactivity', () => {
 			await expect( overlayMenuFirstElement ).toBeFocused();
 
 			// Test: overlay menu traps focus
-			await pageUtils.pressKeys( 'Tab', { times: 2, delay: 50 } );
-			await expect( closeMenuButton ).toBeFocused();
-			await pageUtils.pressKeys( 'Shift+Tab', { times: 2, delay: 50 } );
-			await expect( overlayMenuFirstElement ).toBeFocused();
+			/**
+			 * @todo Determine if focus trapping is required. If not, these assertions
+			 * can be removed and seemingly the next test too — see its comment.
+			 */
+			// await pageUtils.pressKeys( 'Tab', { times: 2, delay: 50 } );
+			// await expect( closeMenuButton ).toBeFocused();
+			// await pageUtils.pressKeys( 'Shift+Tab', { times: 2, delay: 50 } );
+			// await expect( overlayMenuFirstElement ).toBeFocused();
```

Comment:
This tests focus wrapping, i.e. the focus moving either to the first tabbable to the last or vice versa. Since this PR proposes removing the logic for this, focus doesn’t wrap. Any items outside the modal cannot be focused. I think that suffices and seems more accessible because focus isn’t trapped within the document by wrapping it. I just wanted to highlight this in case someone thinks differently.

I’ll go ahead and push a commit that removes this part of the test.